### DISCUSSION TITLE
Labgym used for other species

### DISCUSSION THREAD

**Discussion by HappyDomi** [umyelab/LabGym#66] [Q&A]
Hi, 
I was wondering if Labgym has been used for other species, or it it's possible to do so ? 

I've mostly seen discussions about rats or other small species. 

I'm asking this because I would like to try it for the behaviour if cows and calves. So before diving in the technical aspects I would like to know if it was possible. 

Thank you, it's really new for me, but it seems really promising  !

**Comment by yujiahu415** [comment]
Hi,
LabGym does not have restrictions on animal species. So it can be used for cows and calves.
But let me know if you have any difficulties.

**Reply by HappyDomi** [reply]
Okay thanks, I'll try it and let you know if I need !

**Reply by yujiahu415** [reply]
Absolutely!
